src/noisepy/seis/pnwstore.py

```diff
@@ -17,6 +17,19 @@
 logger = logging.getLogger(__name__)
 
 
+def channel_filter(stations: List[str], ch_prefix: str) -> Callable[[Channel], bool]:
```

Comment:
This appears to be a generalization of the `channel_filter` in `scedc_s3store.py`. If that's the case then let's update that one to avoid duplicating the code.

Reply:
Agreed. I removed this one and changed the one in SCEDC store.